Unable to build site locally

Experiencing the following errors trying to build the website locally:

```
Error: Error building site: "/home/cdaley/go/src/github.com/shipwright-io/website/content/en/blog/posts/2020-10-21-intro-shipwright-pt1/index.md:34:1": failed to extract shortcode: template for shortcode "imgproc" not found
```
```
Error: Error building site: "/home/cdaley/go/src/github.com/shipwright-io/website/content/en/_index.html:7:1": failed to extract shortcode: template for shortcode "blocks/cover" not found
```

```
cdaley@redhat [~/go/src/github.com/shipwright-io/website] (main)
$ hugo version
Hugo Static Site Generator v0.74.3-DA0437B4/extended linux/amd64 BuildDate: 2020-07-23T16:30:30Z
cdaley@redhat [~/go/src/github.com/shipwright-io/website] (main)
$ make build
rm -rf public/*
hugo -t docsy --minify
Total in 12 ms
Error: Error building site: "/home/cdaley/go/src/github.com/shipwright-io/website/content/en/_index.html:7:1": failed to extract shortcode: template for shortcode "blocks/cover" not found
make: *** [Makefile:7: build] Error 255
```

--- a/README.md
+++ b/README.md
@@ -18,7 +18,7 @@ To use this repository, you need the following installed locally:
 Once installed, clone the repository and navigate to the directory:
 
 ```bash
-$ git clone https://github.com/shipwright-io/website.git
+$ git clone --recurse-submodules --depth 1 https://github.com/shipwright-io/website.git
 $ cd website
 ```
 

